### ISSUE TITLE
Isn't possible to configure different traffic label on the same physical network when creating a new zone

### ISSUE THREAD

**Issue by msinhore** [apache/cloudstack#5733]
##### ISSUE TYPE

 * Bug Report
 

##### COMPONENT NAME

~~~
UI
~~~

##### CLOUDSTACK VERSION

~~~
4.16.0
~~~

##### OS / ENVIRONMENT

Management server: CentOS 7
Browser used: Firefox

##### SUMMARY

During the Zone Creation, isn't possible to set different traffic labels.  

##### STEPS TO REPRODUCE

~~~
In the UI, select Infrastructure/Zone and then, Add Zone. In the Physical Network step, when changing the traffic label of one element will changed it of all elements. 
~~~

##### EXPECTED RESULTS

~~~
Different traffic label for each element.
~~~

##### ACTUAL RESULTS

~~~
When changing the traffic label of one element will changed it of all elements. 
~~~

**Comment by utchoang** [comment]
@msinhore I think this issue is similar to issue https://github.com/apache/cloudstack/issues/5705 and will be fixed in PR https://github.com/apache/cloudstack/pull/5714

**Comment by msinhore** [comment]
@utchoang ok, I will close this issue as it was reported in #5714

**Comment by msinhore** [comment]
@utchoang ok, I closed this issue as it was reported in #5714
